assets/src/edit-story/components/header/buttons/buttonWithChecklistWarning.js

```diff
@@ -46,15 +52,25 @@ function ButtonWithChecklistWarning({ text, ...buttonProps }) {
     : null;
 
   const button = (
-    <Primary onPointerEnter={refreshChecklist} {...buttonProps}>
-      <ButtonContent>
-        {text}
-        {tooltip && <WarningIcon />}
-      </ButtonContent>
-    </Primary>
+    <Button
+      variant={BUTTON_VARIANTS.RECTANGLE}
+      type={BUTTON_TYPES.PRIMARY}
+      size={BUTTON_SIZES.SMALL}
+      onPointerEnter={refreshChecklist}
+      {...buttonProps}
+    >
+      {text}
+      {tooltip && <Icons.ExclamationTriangle height={24} width={24} />}
```

Comment:
Do we still want an alert icon here? The [design](https://www.figma.com/file/NHHMwIfxMnz39NxqkrFb7R/Stories---Post-Stable?node-id=2172%3A244419) doesn't have one anymore... And if we do, should we use the circular one rather than the triangle one - given that the triangle is used for warnings and the circle for errors.

Also, should the tooltip be laid out as in the design?
![Screen Shot 2021-02-15 at 16 13 59](https://user-images.githubusercontent.com/637548/107993674-ff19c580-6fa8-11eb-91eb-6912d00d547e.png)

Finally, the designs mention that the user can click on the icon to go to the checklist, but not sure how that's going to work. Maybe we just want the tooltip to come with a clickable link, but that might also be tricky?

Reply:
Also, using the boolean'ess of `tooltip` to toggle the icon seems confusing. Could we add a boolean variable instead?

Reply:
> Do we still want an alert icon here?

IDK, it existed before but perhaps not anymore 🤷  I think this can be done separately though once it has been confirmed -- the whole pre-publishing adjustment part and the tooltip with link, we can perhaps for now keep that part as it was, WDYT?

Reply:
Create a P2 follow-up ticket to sort that stuff out. Makes sense to delay until layer, as it's far from trivial and probably not that important.